spec/features/course/assessment/question/text_response_management_spec.rb

```diff
@@ -83,13 +83,10 @@
           find('a.add_fields', text: link).trigger('click')
           within all('.edit_question_text_response '\
             'tr.question_text_response_solution')[i] do
-            # A custom css selector, :last is added here because +fill_in_rails_summernote+ doesn't
-            # acknowledge the scope defined by capabara.
-            # This works only if +click_link+ is executed before each option.
-            fill_in_rails_summernote '.question_text_response_solutions_solution:last',
-                                     solution[:solution]
+            find('input.text-response-solution').set solution[:solution]
```

Comment:
Summernote editor was removed for text response solutions, so can just use set()